src/components/Popup/Popup.tsx

```diff
@@ -160,41 +148,22 @@ export default class Popup extends UIComponent<Extendable<IPopupProps>, IPopupSt
   private renderPopperChildren = (
     rtl: boolean,
     accessibility: IAccessibilityBehavior,
-    { ref, style }: PopperChildrenProps,
+    { ref, style: popupPlacementStyles, placement }: PopperChildrenProps,
   ) => {
-    const { basic, content } = this.props
-    const computedStyle = rtl ? rtlCSSJS(style) : style
+    const { content } = this.props
 
     return (
-      <Popup.Content
-        innerRef={ref}
-        basic={basic}
-        {...rtl && { dir: 'rtl' }}
-        styles={computedStyle}
-        {...accessibility.attributes.popup}
-        {...accessibility.keyHandlers.popup}
-      >
-        {content}
-      </Popup.Content>
+      <Ref innerRef={domElement => ref(domElement)}>
+        <Popup.Content
+          {...rtl && { dir: 'rtl' }}
+          style={popupPlacementStyles}
+          {...accessibility.attributes.popup}
+          {...accessibility.keyHandlers.popup}
+          data-placement={placement}
```

Comment:
why is this needed? accessibility / w3c standards? it's not adding any functionality

Reply:
these lines reflect changes that were merged by another PR - please, take a look on the current state of master. So, these ones just project these to the changes made in this PR

Reply:
I was talking about line 162:
````typescript
data-placement={placement}
```

Reply:
oh, got it - let me address..